packages/theming/apple-theme/src/appleTypography.ios.ts

```diff
@@ -1,8 +1,5 @@
-import globalTokens from '@fluentui-react-native/design-tokens-ios/light/tokens-global.json';
-
 import { FontSize, FontSizes, FontWeightValue, Typography, Variants } from '@fluentui-react-native/theme-types';
-
-const fontTokens = globalTokens.font;
+import { createFontAliasTokens } from './createFontAliasTokens.ios';
```

Comment:
I don't think we need the .ios here

Reply:
For now we don't, but this does fit the pattern of [other imports](https://github.com/microsoft/fluentui-react-native/blob/8466c50ff1204515570e59456d6428bdd6a933ff/packages/theming/apple-theme/src/appleTheme.ios.ts#L2-L4) in this package, so I say let's keep it this way.